os/customizing-sshd.md

```diff
@@ -41,38 +41,73 @@ systemd:
       enable: true
       contents: |
         [Socket]
-        ListenStream=2222
+        ListenStream=222
         FreeBind=true
         Accept=yes
         [Install]
         WantedBy=multi-user.target
 ```
 
-`sshd` will now listen only on port 2222 on all interfaces when the system is built.
+`sshd` will now listen only on port 222 on all interfaces when the system is built.
+
+### Disabling socket-activation for sshd
+
+It may be desirable to disable socket-activation for sshd to ensure it will reliably accept connections even when systemd or dbus aren't operating correctly.
+
+To configure sshd on Container Linux without socket activation, a Container Linux Config file similar to the following may be used:
+
+```yaml container-linux-config
+systemd:
+  units:
+  - name: sshd.socket
+    mask: true
+  - name: sshd.service
+    enable: true
+    contents: |-
```

Comment:
After looking properly at the unit we ship, I have no clue why we're overwriting it.

I'm pretty sure I started overwriting it just because our existing docs said to and never checked properly.

With fresh eyes, it looks like masking `sshd.socket` is sufficient. I'll update it to that.